### ISSUE TITLE
Data in Meteor Js application is kept in local mongodb. I can't migrate to cloud mongodb.

### ISSUE THREAD

**Issue by zehrademirel** [meteor/meteor#11805]
Hello, I have developed an application in meteor js. Data is stored in local mongodb. I can't connect to cloud mongodb. I am using windows computer and powershell.

**Comment by denihs** [comment]
Hi,

What's the problem exactly? Are you getting any errors? 

Are you trying to copy the data from your local database or just trying to connect your app to your cloud MongoDB?

If it's an error that you're getting, we need a reproduction. [See more on that](https://github.com/meteor/meteor/blob/devel/CONTRIBUTING.md#reporting-a-bug-in-meteor).

**Comment by zehrademirel** [comment]
I want to connect my application to cloud mongodb.
![image](https://user-images.githubusercontent.com/62859366/145953902-66b67cfc-545f-4e0f-9612-3b7cc44d1864.png)

**Comment by zehrademirel** [comment]
I want to connect my application to cloud mongodb.
![image](https://user-images.githubusercontent.com/62859366/145954049-2df34659-31d4-45ac-ad61-69c72061a149.png)

**Comment by zehrademirel** [comment]
I solved the problems by doing research. Thank you. I upgraded the current versions. 
I first fixed the expired certificates issue due to updates as I am using Powershell. ( $Env:NODE_TLS_REJECT_UNAUTHORIZED=0 ) I added the parentheses.

I deleted or updated the packages that were causing the problem.

meteor npm install --save

Then I did the process of moving the meteor to the atlas.

$Env:MONGO_URL="mongodb+srv://zehrademirel:password@cluster0.mfntl.mongodb.net/dbName?retryWrites=true&w=majority"

I then ran it.
meteor
